fury/ui.py

```diff
@@ -3034,10 +3034,12 @@ def __init__(self, label, position=(0, 0), font_size=18):
             the button of the option.
         font_size : int
             Font size of the label.
+        checked : bool
```

Comment:
`bool, optional` instead. You can update the previous parameter in the docstring too. Thank you

Reply:
"update the previous parameter in the docstring"
Do you mean adding "optional" whenever there are default values?  That means a lot of these attributes need to be changed, 
```python
def __init__(self, label, position=(0, 0), font_size=18, checked=False):
        """
        Parameters
        ----------
        label : str
            Text to be displayed next to the option's button.
        position : (float, float), optional
            Absolute coordinates (x, y) of the lower-left corner of
            the button of the option.
        font_size : int, optional
            Font size of the label.
        checked : bool, optional
            Boolean value indicates the initial state of the option
        """
```

Reply:
Ok, you are right, we need a specific PR  for this. Just do it for you case